frontend/packages/shared/src/components/InputActionWrapper/InputActionWrapper.module.css

```
@@ -0,0 +1,5 @@
+.buttons {
+  margin-top: var(--fds-spacing-3);
+  display: flex;
+  align-items: center;
+}
```

Should we add a minimum height to ensure that the UI does not jump when the action buttons became visible on hover? 🤔 😄 
```suggestion
.buttons {
  margin-top: var(--fds-spacing-3);
  display: flex;
  align-items: center;
  /* ensure to have same height as action buttons to avoid jumping UI */
  min-height: var(--fds-component-mode-height-medium);
}
```